analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -199,6 +211,26 @@ class GoMod(
         return graph.nodes().filterTo(mutableSetOf()) { it.name in vendorModuleNames }
     }
 
+    /**
+     * Return the module names of all transitive main module dependencies. This excludes test-only dependencies.
+     */
+    private fun getTransitiveMainModuleDependencies(projectDir: File): Set<String> {
+        val result = mutableSetOf<String>()
+
+        val list = run(
+            "list", "-deps", "-f", "{{with .Module}}{{.Path}} {{.Version}}{{end}}", "./...", workingDir = projectDir
+        )
+
+        list.stdout.lines().forEach { line ->
+            val columns = line.split(' ')
+            if (columns.size != 2) return@forEach
+
+            result += columns[0]
+        }
+
+        return result
```

Comment:
You can make it a bit more compact:
```kotlin
return list.stdout.lineSequence()
    .map { it.split(' ') }
    .filter { it.size == 2 }
    .map { it[0] }
```

Reply:
I believe as-is is more consistent with local style. So in this commit I prefer to keep it. Fine?